byway/src/main/java/com/google/sps/servlets/PlacesServlet.java

```diff
@@ -52,4 +60,39 @@ private String formatLocation(PlaceType location) {
     place = place.replace('_', ' ');
     return place;
   }
+
+  /**
+   * Temporary setup to find entity with hard-coded id value. Set their interests.
+   */
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Just have datastore as a private final variable instead of passing it as a param.

Reply:
Fixed